contracts/registry/DAOFeeRegistry.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: BlueOak-1.0.0
+pragma solidity 0.8.19;
+
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+
+contract DAOFeeRegistry is Ownable {
```

Comment:
i think there are a couple of open discussion points that we should support so as to be as generic and flexible as possible:
- fee's should be applied on a per-rtoken basis
- fee's can be applied to both rsr and rtoken revenue

Reply:
The fee is applied to _all_ revenue but is paid in RSR, that's how it's configured right now. We don't want the veRSR DAO to be holding RTokens.

Reply:
I was also thinking it would be okay to only do RSR, but I can see why we'd want to have generality